### DISCUSSION TITLE
Allow displaying current mod properties (e.g. current speed rate) during gameplay

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#14066] [Ideas]
if we turned on the wind up mod, then we can show that the map speed has changed from 1.00x to 1.03x, etc.

For a flashlight, you can show the field of view (1x -> 0.75x for example)

Such notifications can be shown anywhere in the corner of the screen.

**Comment by Moltanicaa** [comment]
The relevance of the request has grown with the introduction of the "Adaptive speed" mod. It's very useful to see exactly what speed you're currently playing, because many players (myself included) have a very good sense of the original bpm of a song and need to react quickly to changes in speed, especially up.

I'm also not sure if the multiplayer tag is set correctly.

**Comment by peppy** [comment]
I think the path forward that makes sense here is allowing more customisation over the mod display (that currently fades out when not watching a replay). It should give similar flexibility to other components, which will give the user the choice of when they display (always / during show hud / during break), along with some settings for how verbose the display is.
